Competition2018/src/main/java/competition/subsystems/offboard/packets/CommandFinishedPacket.java

```diff
@@ -0,0 +1,19 @@
+package competition.subsystems.offboard.packets;
+
+public class CommandFinishedPacket {
+    public final int commandId;
+    public final int statusCode;
+    
+    private CommandFinishedPacket(byte[] packetData) {
+        if (packetData.length != 2) {
+            throw new IllegalArgumentException();
+        }
+        
+        this.commandId = packetData[0] & 0xFF;
```

Comment:
Some short documentation about the purpose of `0xFF` would be useful

Reply:
For reference — the short answer is _because Java_. https://stackoverflow.com/questions/7401550/how-to-convert-int-to-unsigned-byte-and-back